catroid/src/catroid/java/org/catrobat/catroid/common/FlavoredConstants.java

```diff
@@ -25,15 +25,21 @@
 
 import android.os.Environment;
 
+import org.catrobat.catroid.CatroidApplication;
+
 import java.io.File;
 
 import static org.catrobat.catroid.common.Constants.MAIN_URL_HTTPS;
 
 public final class FlavoredConstants {
 
-	public static final File DEFAULT_ROOT_DIRECTORY = new File(
+	public static final File DEFAULT_ROOT_DIRECTORY = CatroidApplication.getAppContext().getFilesDir();
+
+	public static final File OLD_EXTERNAL_STORAGE_ROOT_DIRECTORY = new File(
```

Comment:
why would you call it `OLD_EXTERNAL_STORAGE_DIRECTORY`? if there is no new one. I think it should be fine if you just called it `EXTERNAL_STORAGE_DIRECTORY`.